clearpath_control/launch/control.launch.py

```diff
@@ -31,20 +31,84 @@
 # Redistribution and use in source and binary forms, with or without
 # modification, is not permitted without the express permission
 # of Clearpath Robotics.
-
 from launch import LaunchDescription
-from launch.actions import DeclareLaunchArgument, GroupAction
-from launch.conditions import UnlessCondition
-from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch.actions import DeclareLaunchArgument, GroupAction, OpaqueFunction
+from launch.conditions import IfCondition, UnlessCondition
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
 from launch_ros.actions import Node
-from launch_ros.parameter_descriptions import ParameterValue
+from clearpath_config.common.utils.dictionary import unflatten_dict
+from clearpath_config.common.utils.yaml import read_yaml
 
+REMAPPINGS = [
+    ('joint_states', 'platform/joint_states'),
+    ('dynamic_joint_states', 'platform/dynamic_joint_states'),
+    ('platform_velocity_controller/odom', 'platform/odom'),
+    ('platform_velocity_controller/cmd_vel_unstamped', 'platform/cmd_vel_unstamped'),
+    ('/diagnostics', 'diagnostics'),
+    ('/tf', 'tf'),
+    ('/tf_static', 'tf_static'),
+    ('~/robot_description', 'robot_description'),
```

Comment:
Should this be relative?

Reply:
To be honest, I am not sure why we have it setup this way. I just kept it the way we had it before: https://github.com/clearpathrobotics/clearpath_common/blob/humble/clearpath_control/launch/control.launch.py.